[2024-03-09 03:00:29,085] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting_project492.merge_data 2024-03-09T03:00:15.034586+00:00 [queued]>
[2024-03-09 03:00:29,095] {taskinstance.py:903} INFO - Dependencies all met for <TaskInstance: meeting_project492.merge_data 2024-03-09T03:00:15.034586+00:00 [queued]>
[2024-03-09 03:00:29,095] {taskinstance.py:1095} INFO - 
--------------------------------------------------------------------------------
[2024-03-09 03:00:29,095] {taskinstance.py:1096} INFO - Starting attempt 1 of 1
[2024-03-09 03:00:29,096] {taskinstance.py:1097} INFO - 
--------------------------------------------------------------------------------
[2024-03-09 03:00:29,107] {taskinstance.py:1115} INFO - Executing <Task(PythonOperator): merge_data> on 2024-03-09T03:00:15.034586+00:00
[2024-03-09 03:00:29,111] {standard_task_runner.py:52} INFO - Started process 2255 to run task
[2024-03-09 03:00:29,113] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'meeting_project492', 'merge_data', '2024-03-09T03:00:15.034586+00:00', '--job-id', '367', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/first_dag.py', '--cfg-path', '/tmp/tmpz9we9pb9', '--error-file', '/tmp/tmphujx3sv2']
[2024-03-09 03:00:29,116] {standard_task_runner.py:77} INFO - Job 367: Subtask merge_data
[2024-03-09 03:00:29,150] {logging_mixin.py:109} INFO - Running <TaskInstance: meeting_project492.merge_data 2024-03-09T03:00:15.034586+00:00 [running]> on host 62b9685fb4d1
[2024-03-09 03:00:29,197] {taskinstance.py:1254} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=meeting_project492
AIRFLOW_CTX_TASK_ID=merge_data
AIRFLOW_CTX_EXECUTION_DATE=2024-03-09T03:00:15.034586+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-03-09T03:00:15.034586+00:00
[2024-03-09 03:00:29,228] {python.py:151} INFO - Done. Returned value was: None
[2024-03-09 03:00:29,237] {taskinstance.py:1219} INFO - Marking task as SUCCESS. dag_id=meeting_project492, task_id=merge_data, execution_date=20240309T030015, start_date=20240309T030029, end_date=20240309T030029
[2024-03-09 03:00:29,286] {local_task_job.py:151} INFO - Task exited with return code 0
[2024-03-09 03:00:29,310] {local_task_job.py:261} INFO - 0 downstream tasks scheduled from follow-on schedule check
